reference/7.0/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,506 @@
+----            ------
+README.md         3355
+CONTRIBUTING.md    247
+
+   Type: Automation
+
+Name                      Length
+----                      ------
+.openpublishing.build.ps1    796
+build.ps1                   7495
+ci.yml                       645
+ci-steps.yml                2035
+daily.yml                   1271
+```
+
+### Format-Wide
+
+The `Format-Wide` cmdlet allows you to display the value one property for
+objects in a collection as a multi-column list.
+
+For this example, we want to see the filename and the size (in kilobytes) as a
+wide listing. Since `Format-Wide` does not display more than one property, we
+use a calculated property to combine the value of two properties into a single
+value.
+
+```powershell
+Get-ChildItem -File |
+  Format-Wide -Property @{e={'{0} ({1:N2}kb)' -f $_.name,($_.length/1kb)}}
+```
+
+```Output
+.editorconfig (0.18kb)                          .gitattributes (0.41kb)
+.gitignore (0.22kb)                             .localization-config (0.23kb)
+.markdownlint.json (2.40kb)                     .openpublishing.build.ps1 (0.78kb)
+.openpublishing.publish.config.json (2.25kb)    .openpublishing.redirection.json (324.60kb)
+build.ps1 (7.32kb)                              ci.yml (0.63kb)
+ci-steps.yml (1.99kb)                           CONTRIBUTING.md (0.24kb)
+daily.yml (1.24kb)                              LICENSE (18.60kb)
+LICENSE-CODE (1.08kb)                           README.md (3.28kb)
+ThirdPartyNotices (1.20kb)
+```
+
+### Group-Object
+
+The `Group-Object` cmdlet displays objects in groups based on the value of a
+specified property. In this example we are using a calculated property to count
+the number of files of each content type.
+
+```powershell
+Get-ChildItem -File |
+  Sort-Object extension |
+    Group-Object -NoElement -Property @{
+      name='Type'
+      expression={
+        switch ($_.extension) {
+          '.md'   {'Content'}
+          ''      {'Metacontent'}
+          '.ps1'  {'Automation'}
+          '.yml'  {'Automation'}
+          default {'Configuration'}
+        }
+      }
+    }
+```
+
+```Output
+Count Name
+----- ----
+    5 Automation
+    7 Configuration
+    2 Content
+    3 Metacontent
+```
+
+### Measure-Object
+
+The `Measure-Object` cmdlet calculates the numeric properties of objects. In
+this example, we use a calculated property to get the count (**Sum**) of the
+numbers, between 1 and 10, that are evenly divisible by 3.
+
+```powershell
+1..10 | Measure-Object -Property {($_ % 3) -eq 0} -Sum
+```
+
+```Output
+Count             : 10
+Average           :
+Sum               : 3
+Maximum           :
+Minimum           :
+StandardDeviation :
+Property          : ($_ % 3) -eq 0
+```
+
+> [!NOTE]
+> Unlike the other cmdlets, `Measure-Object` does not accept a hashtable for
+> calculated properties. You must only use a script block.
+
+### Select-Object
+
+You can use calculated properties to add additional members to the objects
```

Comment:
```suggestion
You can use calculated properties to add additional members to the object's
```

Reply:
It is not a possessive case. I am talking about the objects being output. Not the output from the object.